src/plugins/formatters.commerce.ts

```diff
@@ -14,9 +14,11 @@ import { parseDecimal } from './util.i18n';
 // Template imports
```

Comment:
The SubscriptionPriceFormatter logic was ported from the corresponding [formatter](https://github.com/Squarespace/template-compiler/blob/main/core/src/main/java/com/squarespace/template/plugins/platform/CommerceFormatters.java/#L473-L533) in template-compiler